index.js

```diff
@@ -401,3 +401,83 @@ async function syncDiscordUser(discordId, cavUser = null) {
     await discordProfile.roles.add(rolesToSet)
         .catch(console.error);
 }
+
+/*
+    REACT TO ROLES || START
+*/
+
+bot.on('raw', event => {
+    const eventName = event.t;
+
+    if(eventName === 'MESSAGE_REACTION_ADD') {
+        if(event.d.message_id === cfg.React.message_id) {
+            var reactionChannel = bot.channels.get(event.d.channel_id);
```

Comment:
This line is duplicated in the else if down below. We should move it above the conditions, so that we only write it once